client/components/empty-content/style.scss

```diff
@@ -47,11 +47,19 @@
 }
 
 .empty-content__action {
-	margin: 0 0 10px 10px;
+	margin: 0 0 10px 0;
```

Comment:
It looks like this wasn't working because this style rule has the same specificity as `.button` general style, and is overwritten in the cascade loading order. 

We should probably change the selector to add 1 more level of specificity, so other styles are not overwritten unexpectedly as well. That would allow us to avoid the extra first-child/last-child logic.

Aside - your current styles seem to render a spacing similar to if this margin were also changed to `0 10px 10px`,  which I think also does look better than when that original margin rule is actually active.

Reply:
That's a good point, I should have taken a much simpler approach. I just pushed 19fa36d020161d78c7103e32ee9bc6f4013f3179 to address that.

> your current styles seem to render a spacing similar to if this margin were also changed to 0 10px 10px (i.e. short for 0 10px 10px 10px), which I think also does look better than when that original margin rule is actually active.

Since I reverted my changes we're back at `0 0 10px 10px` ; I figured I would leave it like that now.